test/integ/pandas/test_arrow_pandas.py

```diff
@@ -14,10 +14,10 @@
 import pytest
 
 try:
-    from snowflake.connector.constants import TABLE_UNIT
+    from snowflake.connector.constants import IterUnit
 except ImportError:
     # This is because of olddriver tests
-    TABLE_UNIT = "table"
+    IterUnit.TABLE_UNIT = "table"
```

Comment:
This isn't enough, do something like
```suggestion
    class IterUnit(enum.Enum):
        ROW_UNIT = "row"
        TABLE_UNIT = "table"
```
Just make sure to import `enum`

Reply:
Odd, I added this before re-requesting review, seems like it didn't successfully push.